package.json

```diff
@@ -13,6 +13,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@arranger/server": "file:../arranger/modules/server",
```

Comment:
placeholder until arranger is updated: https://github.com/overture-stack/arranger/pull/572